lib/discover.js

```
@@ -79,18 +79,17 @@ TesselSeeker.prototype.start = function(opts) {
           self.emit('tessel', tessel);
         } else {
           debug('Fetching name:', tessel.connection.host);
-          tessel.getName()
+          return tessel.getName()
```

Also shouldn't change anything but at least we don't swallow the error